### PR TITLE
Feature/my past session

### PR DESCRIPTION
**Pull request by Nituyadav0109**

### CODE REVIEW

**Review by Sparsh79 — CHANGES_REQUESTED** [review]
Please remove the test output folder. Ideally it should have been ignored via gitignore.

**Comment by Nituyadav0109** [comment]
> Please remove the test output folder. Ideally it should have been ignored via gitignore.

Removed

**Review by Sparsh79 — CHANGES_REQUESTED** [review]
Please revert back the changes in package-lock.json file. Rest looks fine to me.

File: go1percent/page-objects/myPastSession/myPastSessionpages.js
Comment (Ritika-gupta20) [inline go1percent/page-objects/myPastSession/myPastSessionpages.js:1]: Why are we using xpath here?

File: go1percentTS/package.json
Comment (Ritika-gupta20) [inline go1percentTS/package.json:1]: please remove this

File: go1percentTS/nightwatch.conf.js
Comment (Ritika-gupta20) [inline go1percentTS/nightwatch.conf.js:1]: please remove this file

File: go1percentTS/package.json
Comment (Nituyadav0109) [reply go1percentTS/package.json:1]: Removed package.json file

File: go1percentTS/nightwatch.conf.js
Comment (Nituyadav0109) [reply go1percentTS/nightwatch.conf.js:1]: Removed nightwatch.conf.js file

**Review by Sparsh79 — APPROVED** [review]

**Review by Ritika-gupta20 — APPROVED** [review]

**Pull request merged**